src/sidebar/media-embedder.js

```diff
@@ -396,17 +396,12 @@ function replaceLinkWithEmbed(link) {
  *     the width of the embed.
  */
 export function replaceLinksWithEmbeds(element, { className } = {}) {
-  let links = element.getElementsByTagName('a');
+  // Get a static (non-live) list of <a> children of `element`.
+  // It needs to be static because we may replace these elements as we iterate over them.
+  const links = Array.from(element.getElementsByTagName('a'));
```

Comment:
There is no functional change here. I've just rewritten the logic in a more modern way to avoid a spurious TS error.